arangod/VocBase/Methods/Collections.cpp

```diff
@@ -237,9 +237,9 @@ VPackBuilder createCollectionProperties(
 
     bool addUseRevs = ServerState::instance()->isSingleServerOrCoordinator();
     bool useRevs =
-        vocbase.server()
-            .getFeature<arangodb::EngineSelectorFeature>()
-            .isRocksDB() &&
+        !info.properties
+             .get(arangodb::StaticStrings::UsesRevisionsAsDocumentIds)
+             .isFalse() &&
```

Comment:
This would change the behavior.
For example, if the attribute is not present at all, `!attribute.get(...).isFalse()` will return true, but the version you are suggesting will return false.
The intention is: for every value that is not identical to `false`, we set the value to true. So if you don't specify the attribute, we set the value to true as well. Not just when the attribute has a value of `true`.